standalone-metastore/pom.xml

```diff
@@ -31,7 +31,7 @@
   </modules>
   <properties>
     <hive.version>4.0.0-SNAPSHOT</hive.version>
```

Comment:
What about this? It doesn't require changing?

Reply:
Yeah, I needed to fix it in several more places 😢
I hope I am finished for now 😄